### PR TITLE
modified ARBaseLib to provide local broadcasts for cameraPreview fram…

### PR DESCRIPTION
**Pull request by Kartik-RTP**
modified ARBaseLib to provide local broadcasts for cameraPreviewFrame data which might be neeeded by someone in their app like me

### CODE REVIEW

**Comment by Kartik-RTP** [comment]
I created a new Folder in AndroidStudioProjects folder to contain the modified ARBaseLib...rest nothing has been changed by me ...so other conflicts can be resolved by keeping the files from latest master commit

File: AndroidStudioProjects/ARSimpleNativeCarsProj/aRSimpleNativeCars/src/main/jni/Android.mk [hunk 1]
```diff
@@ -67,6 +67,7 @@ define add_shared_module
 	LOCAL_SRC_FILES:=lib$1.so
 	include $(PREBUILT_SHARED_LIBRARY)
 endef
+
```
Comment (Neuroforge) [inline AndroidStudioProjects/ARSimpleNativeCarsProj/aRSimpleNativeCars/src/main/jni/Android.mk]: Please revert this change. I have requested a review of these changes for you.

**Comment by Neuroforge** [comment]
Please correct the ~4000 files before committing. We'll only be able to accept the required changes for the resolution of this issue not minor file alterations.

**Comment by Kartik-RTP** [comment]
Hey , I have modified the line endings by making the "* text=auto" in ".gitattributes file" and changed the line endings to "lf" for most cases . Can u please check it ? Also do I have to create a new pull request now?
